charts/argo-events/README.md

```diff
@@ -31,81 +31,135 @@ You can install the CRDs manually from `crds` folder.
 
```

Comment:
We should add an ## Upgrading section inside the README.

And maybe we place a howto in it how users can adopt existing CRDs:

```bash
for crd in "eventbus.argoproj.io" "eventsources.argoproj.io" "sensors.argoproj.io"; do
  kubectl label --overwrite crd $crd app.kubernetes.io/managed-by=Helm
  kubectl annotate --overwrite crd $crd meta.helm.sh/release-namespace=<YOUR_NAMESPACE>
  kubectl annotate --overwrite crd $crd meta.helm.sh/release-name=<YOUR_HELM_REALEASE>
done
```

Reply:
I'm not 100% sure about this - as CRDs are global shared resource I haven't added any labels in these. How should it work in case of namespace install?

Reply:
@mkilchhofer Any update on this?

Reply:
When they live inside the `/templates` directory helm will try to manage them. If these labels/annotations are absent, helm will panic and say that the resource already exists.

These labels and annotations are added on first install normally for all resources living in the /templates directory.

Reply:
@mkilchhofer Added section with Upgrade information that contains your script.